tests/d3d12_crosstest.h

```diff
@@ -539,25 +539,25 @@ static HRESULT create_vkd3d_device(struct vkd3d_instance *instance,
     {
         VK_KHR_DRIVER_PROPERTIES_EXTENSION_NAME,
     };
-    static const struct vkd3d_optional_device_extensions_info optional_extensions =
-    {
-        .type = VKD3D_STRUCTURE_TYPE_OPTIONAL_DEVICE_EXTENSIONS_INFO,
-        .extensions = device_extensions,
-        .extension_count = ARRAY_SIZE(device_extensions),
-    };
 
     struct vkd3d_device_create_info device_create_info;
     VkPhysicalDevice vk_physical_device;
 
     if (!(vk_physical_device = select_physical_device(instance)))
         return E_INVALIDARG;
 
-    memset(&device_create_info, 0, sizeof(device_create_info));
```

Comment:
Why not keep the memset?

Reply:
Oh, I was planning on refactoring the memsets to `(struct StructName) { }` initializers but I decided to do that elsewhere and not in this set these were left over, will fix.

Reply:
Fixed.